wormhole-connect/src/utils/localStorage.ts

```diff
@@ -0,0 +1,56 @@
+import { TransactionLocal } from 'config/types';
+
+// Adds an object to localStorage
+export const addItemToLocalStorage = (
+  id: string, // Key for the item
+  data: object, // Item data
+  prefix: string, // Prefix to use for the item key
+  max?: number, // Max number of items allowed with the same prefix
+) => {
+  const ls = window.localStorage;
+  const items: Array<string> = [];
+
+  // Get all current items with the prefix
+  for (let i = 0; i < ls.length; i++) {
+    const itemKey = ls.key(i);
```

Comment:
> The order of keys is user-agent defined, so you should not rely on it.

https://developer.mozilla.org/en-US/docs/Web/API/Storage/key

Reply:
We can't even depend on a 0-based indexed order 😢 I'll add a timestamp to the localStorage and check that one instead.

Reply:
Scratch that, I'll use a single entry with an array as you mentioned below.